Sources/PackageCollectionsSigning/PackageCollectionSigning.swift

```diff
@@ -22,16 +22,35 @@ public protocol PackageCollectionSigner {
     ///   - collection: The package collection to be signed
     ///   - certChainPaths: Paths to all DER-encoded certificates in the chain. The certificate used for signing
     ///                     must be the first in the array.
-    ///   - certPrivateKeyPath: Path to the private key (*.pem) of the certificate
+    ///   - privateKeyPEM: Data of the private key (*.pem) of the certificate
     ///   - certPolicyKey: The key of the `CertificatePolicy` to use for validating certificates
     ///   - callback: The callback to invoke when the signed collection is available.
     func sign(collection: PackageCollectionModel.V1.Collection,
               certChainPaths: [URL],
-              certPrivateKeyPath: URL,
+              privateKeyPEM: Data,
               certPolicyKey: CertificatePolicyKey,
               callback: @escaping (Result<PackageCollectionModel.V1.SignedCollection, Error>) -> Void)
 }
 
+extension PackageCollectionSigner {
+    public func sign(collection: PackageCollectionModel.V1.Collection,
```

Comment:
We should have docs for this method too. i.e., just copy-paste the old docs here.